services/solr-names-updater/src/solr_names_updater/worker.py

```diff
@@ -66,7 +67,11 @@ def is_names_event_msg_type(msg: dict):
 
 def is_processable(msg: dict):
     """Determine if message is processable using message type of msg."""
-    if msg and is_names_event_msg_type(msg):
+    nr_num = msg.get('nrNum', None)
+    nr = RequestDAO.find_by_nr(nr_num)
```

Comment:
`nr = RequestDAO.find_by_nr(nr_num)` can be removed here as the `if` block will have it

Reply:
done